src/sass/base/_colors.scss

```diff
@@ -1,13 +1,13 @@
 body {
-    background-color: white;
-    padding: 0;
     margin: 0;
+    background-color: $white;
```

Comment:
I doubt this is better than the status quo.

Ideally we would use a descriptive name for every color that depends on the context. This is also being done in #9.

For now, I'd disable the lint with the goal of re-enabling it (and actually cleaning up colors, not using `$white`) oncce the themeing is sorted out.